src/app/app.component.html

```diff
@@ -1,13 +1,12 @@
 <nav>
   <mat-toolbar>
-    <span class="brand">Prylabs</span>
-
-    <a routerLink="/" class="mat-button">Home</a>
-    <a routerLink="/stats" class="mat-button">Stats</a>
-    <a routerLink="/participate" class="mat-button">Participate</a>
+    <span class="brand">Prysm Eth2 Testnet</span>
```

Comment:
I think this is going to be way too crowded on mobile. Please confirm

Reply:
![image](https://user-images.githubusercontent.com/7246818/56815563-a8f5f400-680f-11e9-8ecc-1e7968081548.png)


Mobile is going to be the majority of the traffic. It must be usable.